packages/client/src/forms/configuration/formConfig/reducer.ts

```diff
@@ -92,6 +95,74 @@ function getReadyState({ formDrafts, questionConfig }: IFormConfig) {
   }
 }
 
+function shiftUp(fieldMap: IConfigFieldMap, fieldId: string) {
```

Comment:
I'm quite confused about what this function does. Maybe it would be a good idea to discuss it in detail